packages/jaeger-ui/src/components/SearchTracePage/SearchResults/DownloadResults.tsx

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2023 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as React from 'react';
+import {Button} from 'antd';
+
+type Props = {
+    onDownloadResultsClicked: () => void;
+};
+
+
+export default function DownloadResults(props: Props) {
+    const {onDownloadResultsClicked} = props;
+    const downloadBtn = (
+        <Button className="ub-ml2" htmlType="button" onClick={onDownloadResultsClicked}>
+            Download Results
+        </Button>
+    );
+    return (
+        <>
```

Comment:
what does `<>` represent? Is it needed?

Reply:
**<> </>**  
This is called the Empty tag in React. This is a shorter syntax for React Fragments.
Documentation: https://react.dev/reference/react/Fragment
It is needed. If you remove this tag you can get exception like 

> [AltViewOptions.tsx](react-dom.development.js:14887 Uncaught Error: Objects are not valid as a React child (found: object with keys {downloadBtn})).....

It is used already in this project. See for example AltViewOptions.tsx

Reply:
I was wrong. I got the exception which I described above, but the lint complained about this part of code:
> .../SearchResults/DownloadResults.tsx
[eslint       ]   29:10  error  Fragments should contain more than one child - otherwise, there’s no need for a Fragment at all  react/jsx-no-useless-fragment

I changed the code in a different wait and it works locally without the described error.